### PR TITLE
send email change status order

### PR DESCRIPTION
**Pull request by ghost**
## Description
Send email when change status order
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: resources/views/email/content.blade.php [hunk 1]
```diff
@@ -9,7 +9,8 @@
 <body>
     <div>
         <h1>Welcome To The Website Food-Drink</h1>
-        <p>Hello {{$name}} <span>(email:{{$email}})</span><p>
+        <p>Hello {{$name}}<p>
+        <p>Your email: {{$email}}</p>
```
Comment (at-thaitran) [inline resources/views/email/content.blade.php]: không gửi thêm password à

File: resources/views/email/order-change-status-mail.blade.php [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Mail</title>
+</head>
+<body>
+    <div>
+        <h1>Welcome To The Website Food-Drink</h1>
+        <p>Hello {{$name}}<p>
+        @if($status == $pending)
+            <p>Your orders status has been changed is <span style="color:#F0FF0F">PENDING</span><p>
+        @elseif($status == $accepted)
```
Comment (at-thaitran) [inline resources/views/email/order-change-status-mail.blade.php]: chuyển sang pending không cần gửi mail

Comment (at-dungle) [reply resources/views/email/order-change-status-mail.blade.php]: đang duyệt gửi mail cho ng ta biết cũng được

File: resources/views/email/content.blade.php (hunk 1 above)
Comment (at-dungle) [reply resources/views/email/content.blade.php]: cái này đang gặp lỗi, xử lý sau

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**